docs/a-tour-of-textile.md

```diff
@@ -1349,6 +1453,10 @@ Take a closer look at the "My runs" thread using the `blocks ls` command, which
 {{examples.my_runs.blocks_ls.objc.code}}
 ```
 
+```Swift tab="Swift"
+{{examples.my_runs.blocks_ls.swift.code}}
```

Comment:
even though it's a todo, this needs to be added as an object in the YAML or it wont compile

Reply:
Ah yea I meant to add things to the yml where they were missing, but I'm sure I missed some.

Reply:
Fixed.